src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcMessage.cs

```diff
@@ -19,7 +19,8 @@ internal enum DiagnosticsIpcError : uint
         BadEncoding           = 0x80131384,
         UnknownCommand        = 0x80131385,
         UnknownMagic          = 0x80131386,
-        UnknownError          = 0x80131387
+        UnknownError          = 0x80131387,
+        EnvVarNotFound        = 0x800700CB,
```

Comment:
While this constant was added this [change](https://github.com/dotnet/runtime/commit/d156b9e58d7b09f742d4553bb5a94657cf7e28ae), it doesn't seem to be used. I think a few pieces of code that were originally planned regarding getting a single environment variable still remain in the runtime but are unused. So maybe not add it here for now?